### PR TITLE
MMT-3923: Catch intermittent error from EDL

### PR DESCRIPTION
**Pull request by htranho**
# Overview

### What is the feature?

cmr-graphql returns intermittent error getting acl and groups from EDL.

### What is the Solution?

Catch that error and create banner message to refresh browser. 

### What areas of the application does this impact?

View of single Permission

# Testing

See test session.

### Attachments

N/A

# Checklist

- [x] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by mandyparson** [comment]
Is there no way to reproduce this locally? For testing, all I see is 'see test session'

**Comment by htranho** [comment]
> Is there no way to reproduce this locally? For testing, all I see is 'see test session'

It is about intermittent failing of EDL in production. Couldn't verify this issue on SIT or UAT and also in local mmt instance.

**Review by mandyparson — APPROVED** [review]
Unable to truly test this. But the code checks out.

File: static/src/js/components/ErrorBoundary/ErrorBoundary.jsx [hunk 1]
```diff
@@ -17,9 +17,18 @@ class ErrorBoundary extends React.Component {
   }
 
   static getDerivedStateFromError(error) {
+    let errorMessage = error.message
```
Comment (cgokey) [inline static/src/js/components/ErrorBoundary/ErrorBoundary.jsx:20]: You should add a test that mocks this response

**Review by cgokey — CHANGES_REQUESTED** [review]
I think a test is need that mocks a error response from EDL.

File: static/src/js/components/ErrorBoundary/ErrorBoundary.jsx (hunk 1 above)
Comment (htranho) [reply static/src/js/components/ErrorBoundary/ErrorBoundary.jsx:20]: Added in ErrorBoundary.test.js

**Review by cgokey — APPROVED** [review]

**Pull request merged**